lighthouse-core/test/fixtures/config-plugins/lighthouse-plugin-simple/plugin-simple.js

```diff
@@ -7,12 +7,19 @@
 
 /** @type {LH.Config.Plugin} */
 module.exports = {
+  groups: {
+    'new-group': {
+      title: 'New Group',
+    },
+  },
   audits: [
     {path: 'redirects'},
     {path: 'user-timings'},
   ],
   category: {
     title: 'Simple',
-    auditRefs: [],
+    auditRefs: [
+      {id: 'redirects', weight: 1, group: 'new-group'},
```

Comment:
oh, ha, there weren't any audit refs to add a group to before. Sorry I missed that :)